app/S_game.py

```diff
@@ -0,0 +1,128 @@
+import pygame, sys, random
+from pygame.locals import*
```

Comment:
Change import format. Import with (*) affect the performance.  
```suggestion
from pygame.locals import*
```